src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,80 @@
  */
 
 public class Plane {
+    private String manufacturer;
+    private String model;
+    private int boardNumber;
+    private int passengerCapacity;
+    private int yearOfManufacturing;
 
+    private Plane(PlaneBuilder builder) {
+        this.manufacturer = builder.manufacturer;
+        this.model = builder.model;
+        this.boardNumber = builder.boardNumber;
+        this.passengerCapacity = builder.passengerCapacity;
+        this.yearOfManufacturing = builder.yearOfManufacturing;
+
```

Comment:
what's the purpose of using `this` here?

also, please remove the blank line

Reply:
got it.
fixed